src/modules/highlight/TextHighlighter.ts

```diff
@@ -1876,8 +1888,13 @@ export default class TextHighlighter {
           payload.highlight
         )) as Annotation;
         // if(anno.comment) {
-        if (this.delegate.api && this.delegate.api.selectedAnnotation) {
-          this.delegate.api.selectedAnnotation(anno).then(async () => {});
+        if (
+          this.delegate.annotationModule.config.api &&
+          this.delegate.annotationModule.config.api.selectedAnnotation
+        ) {
+          this.delegate.annotationModule.config.api
+            .selectedAnnotation(anno)
+            .then(async () => {});
```

Comment:
This can be changed to:
```ts
this.delegate.annotationModule.config.api?.selectedAnnotation(anno)
```
I believe, but it's not a huge deal either way

Reply:
yes, this should be now possible, just didn't want to refactor in the first step  too much.

Reply:
i will make this change in a separate iteration, since this is used in more than one place, and more than one module